catalog/ui/catalog-ui-search/src/main/webapp/component/query-custom/query-custom.view.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, setTimeout*/
+// TODO : Possibly Extend query-advanced
+define([
+    'marionette',
+    'underscore',
+    'jquery',
+    './query-custom.hbs',
+    'js/CustomElements',
+    'component/filter-builder-search-form/filter-builder.view',
+    'component/filter-builder-search-form/filter-builder',
+    'js/cql',
+    'js/store',
+    'component/query-settings/query-settings.view',
+    'component/query-advanced/query-advanced.view'
+], function (Marionette, _, $, template, CustomElements, FilterBuilderView, FilterBuilderModel, cql,
+            store, QuerySettingsView, QueryAdvanced) {
+
+    return QueryAdvanced.extend({
+
```

Comment:
Add `template: template,` here or else the `custom-view` handlebars template will never be used. Currently will inherit the template from the extended class.

Reply:
added